base/services/w32time/ntpclient.c

```diff
@@ -44,17 +40,15 @@ InitConnection(PINFO pInfo,
 
     /* Setup server info */
     he = gethostbyname(lpAddress);
-    if (he != NULL)
-    {
-        /* Setup server socket info */
-        ZeroMemory(&pInfo->ntpAddr, sizeof(SOCKADDR_IN));
-        pInfo->ntpAddr.sin_family = AF_INET; // he->h_addrtype;
-        pInfo->ntpAddr.sin_port = htons(NTPPORT);
-        pInfo->ntpAddr.sin_addr = *((struct in_addr *)he->h_addr);
-    }
-    else
+    if (he == NULL)
         return FALSE;
 
+    /* Setup server socket info */
+    ZeroMemory(&pInfo->ntpAddr, sizeof(SOCKADDR_IN));
```

Comment:
@tkreuzer has requested this be sizeof(pInfo->ntpAddr) and this seems reasonable to me.